Formula/m/markdown-oxide.rb

```diff
@@ -0,0 +1,34 @@
+class MarkdownOxide < Formula
+  desc "Personal Knowledge Management System for the LSP"
+  homepage "https://oxide.md"
+  url "https://github.com/Feel-ix-343/markdown-oxide/archive/refs/tags/v0.24.2.tar.gz"
+  sha256 "fac8791c3232fd7407a919bcfdceb76968b0b3d761318e2d5b3af59b453cd1cb"
```

Comment:
is this LSP or an app?

Reply:
LSP

Reply:
I see

Reply:
This is the upstream description, I only expanded PKM.